pyroute2/config/__init__.py

```diff
@@ -4,6 +4,36 @@
 import signal
 import socket
 
+
+class LocalMock:
```

Comment:
Wouldn't an stdlib Mock provide the same functionality ?

Reply:
Yes. 

But there is an unspoken requirement :) The library core must use as little of the stdlib as possible, since it runs sometimes on embedded platforms with limited stdlib. That's why we have `pyroute2-minimal` with `IPRoute` + `netns` functionality only.

An unittest is one of the modules that gets cut when they build embedded platforms.